### PR TITLE
Consolidate SQS message delivery

### PR DESCRIPTION
**Pull request by xbrianh**
This also refactors and re-enables a notify-v2 test

connected to #2171

### CODE REVIEW

File: dss/events/handlers/notify_v2.py [hunk 1]
```diff
@@ -61,22 +61,24 @@ def notify_or_queue(replica: Replica, subscription: dict, metadata_document: dic
...
+                    tombstones.add(bundle_key)
+                elif _bundle_key_regex.match(key):
+                    bundles.add(key)
+            for key in bundles:
+                if key not in tombstones:
+                    mq.send(_format_sqs_message(replica, subscription, "TOMBSTONE", key), delay_seconds=0)
+        else:
+            bundle_key = key.rsplit(".", 1)[0]
+            if not notify(subscription, metadata_document, bundle_key):
+                mq.send(_format_sqs_message(replica, subscription, "TOMBSTONE", bundle_key), delay_seconds=15 * 60)
```
Comment (DailyDreaming) [inline dss/events/handlers/notify_v2.py]: Maybe a dumb question, but why is this set to 15 minutes?

Comment (xbrianh) [reply dss/events/handlers/notify_v2.py]: I would preferred longer, but 15 minutes is the [maximum](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-message-timers.html)

Comment (DailyDreaming) [reply dss/events/handlers/notify_v2.py]: Good to know.

**Comment by xbrianh** [comment]
@kislyuk This PR has been updated to pull in the new SQS functionality from dcplib.

**Pull request merged**